src/parsers/opensea.js

```diff
@@ -6,7 +6,7 @@ export const parse = (asset) => {
     tokenId: asset.token_id,
     collection: _.toLower(asset.asset_contract.address),
     name: asset.name,
-    flagged: !asset.supports_wyvern,
+    flagged: asset.supports_wyvern != null ? !asset.supports_wyvern : undefined,
```

Comment:
i think its better to check if the `supports_wyvern` property exists on the object (and then also not pass it from the indexer)

Reply:
wouldn't `asset.supports_wyvern != null` return true if its undefined?